sale_automatic_workflow/models/account_invoice.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# © 2013 Camptocamp SA (author: Guewen Baconnier)
+# © 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    workflow_process_id = fields.Many2one(
+        comodel_name='sale.workflow.process',
+        string='Sale Workflow Process'
+    )
+    # TODO propose a merge to add this field by default in account module
+    sale_ids = fields.Many2many(
+        comodel_name='sale.order',
```

Comment:
The counterpart field on `sale.order` is a computed field so does this `Many2many` work as expected?

`addons/sale/sale.py`:
```python
     invoice_ids = fields.Many2many("account.invoice", string='Invoices', compute="_get_invoiced", readonly=True, copy=False)
```